sorc/ncep_post.fd/INITPOST_GFS_NEMS_MPIIO.f

```diff
@@ -1111,7 +1111,7 @@ SUBROUTINE INITPOST_GFS_NEMS_MPIIO(iostatusAER)
             enddo
           enddo
           if (me == 0) print*,'sample model pint,pmid' ,ii,jj,l &
```

Comment:
Why only delete ",pmid(ii,jj,l)"? however, the message still has pmid.

Reply:
Good catching. pmid is remove from that printout line.